lib/cmds/space_cmds/generate_cmds/migration.js

```diff
@@ -323,21 +329,27 @@ const generateMigrationScript = async function(environment, contentTypes) {
 
 module.exports.generateMigrationScript = generateMigrationScript
 
-const generateFileName = function(spaceId, environmentId, contentTypeId) {
+const generateFileName = function (spaceId, environmentId, contentTypeId) {
   const contentTypePart = contentTypeId ? `-${contentTypeId}` : ''
 
   return `${spaceId}-${environmentId}${contentTypePart}-${Date.now()}.js`
 }
 
-const createManagementClient = function(managementToken) {
+const createManagementClient = function (managementToken, headers) {
   return createClient({
     accessToken: managementToken,
-    feature: 'migration-generate'
+    feature: 'migration-generate',
+    headers
   })
 }
 
-const getEnvironment = async function(managementToken, spaceId, environmentId) {
-  const client = await createManagementClient(managementToken)
+const getEnvironment = async function (
+  managementToken,
+  spaceId,
+  environmentId,
+  headers
+) {
```

Comment:
I would turn these params into an object now that the argument list is getting long